wiki/Beatmap_search/en.md

```diff
@@ -17,7 +17,7 @@ Beatmap metadata fields can be compared against specific values for more granula
 
 | Comparison | Description |
 | :-: | :-- |
-| `=` or `==` | Equal to |
+| `:` (not in stable), `==` (stable only), or `=` | Equal to |
```

Comment:
maybe `=` first cuz probably most common way to write it

Reply:
the reordering is to avoid "`=`, `==` (stable only), or `:` (not in stable)" being misinterpreted as `=` and `==` only working on stable

can you add a concrete suggestion for how readability would need to be improved?

Reply:
`= (everywhere), == (stable), : (lazer and web)` would be one attempt

Reply:
alright then

```suggestion
| `=` (everywhere), `==` (stable), `:` (lazer and web) | Equal to |
```